config/muxcable.py

```diff
@@ -196,25 +195,53 @@ def mode(state, port, json_output):
 @click.argument('target', required=True, default=None, type=click.INT)
 @click.argument('mode_value', required=True, default=None, type=click.INT)
 @click.argument('lane_map', required=True, default=None, type=click.INT)
-def prbs(port, target, mode_value, lane_map):
+def enable_prbs(port, target, mode_value, lane_map):
 
-    res = y_cable.enable_prbs_mode(port, target, mode_value, lane_map)
+    import sonic_y_cable.y_cable
+    res = sonic_y_cable.y_cable.enable_prbs_mode(port, target, mode_value, lane_map)
     if res != True:
         click.echo("PRBS config unsuccesful")
         sys.exit(CONFIG_FAIL)
     click.echo("PRBS config sucessful")
     sys.exit(CONFIG_SUCCESSFUL)
 
+@muxcable.command()
+@click.argument('port', required=True, default=None, type=click.INT)
+@click.argument('target', required=True, default=None, type=click.INT)
+def disable_prbs(port, target):
+
+    import sonic_y_cable.y_cable
+    res = sonic_y_cable.y_cable.disable_prbs_mode(port, target)
+    if res != True:
+        click.echo("PRBS disable unsuccesful")
+        sys.exit(CONFIG_FAIL)
+    click.echo("PRBS disbale sucessful")
+    sys.exit(CONFIG_SUCCESSFUL)
+
```

Comment:
Maybe you could make `prbs` a group and add `enable` and disable` subcommands to that group?

Reply:
fixed